docs/installation.md

```diff
@@ -0,0 +1,72 @@
+# Getting setup with execution environment support on:
+
+## macOS
+
+### Requirements
+
+- Docker desktop for Mac
+- macOS command line developer tools
+- Internet access (during initial installation)
+
+### Installation using `podman` as the container engine
+
+- Until native source mounts from macOS through the `podman` machine into the execution environment is available, there is not a convenient way to use `podman` on macOS with `ansible-navigator` [Related](https://github.com/containers/podman/issues/8016)
+
+### Installation using `Docker Desktop for Mac` as the container engine
+
+- Install [Docker Desktop for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+- Open a terminal and enter `xcode-select install`, proceed with the command line developer tools installation if prompted
+- Install ansible-navigator `pip3 install ansible-navigator --user`
+- Add the installation path to the PATH (e.g. `echo 'export PATH=$HOME/Library/Python/3.8/bin:$PATH' >> ~/.zshrc`)
+- Refresh the PATH (e.g. `source ~/.zshrc`)
+- Launch ansible-navigator: `ansible-navigator`
+- A one-time download of the demo execution-environment image will happen
```

Comment:
```suggestion
- `ansible-navigator` triggers a one-time download of the demo execution-environment image.
```

Reply:
I'm guessing this is what happens, but in general, you want to show this as a present tense action here and where it is repeated in other procedures below.

Another couple of general comments:
 - Would be nice to number these. Markdown can't do this other than manually saying 1. 2. etc instead of the current bullet list dashes. Since these are long procedures, it does help to have a number so someone can say "it didn't work on step 3' for example.
 - End each one with a period.

Reply:
@samccann I'm not so sure about that. IIRC you can use `1.` for each line and MD will use the right numbers turning it into an ordered list. Plus MyST is much more powerful than CommonMark.

Reply:
@webknjaz could be. I don't write in .md so it could also depend on read-the-docs where this is currently hosted.

Reply:
No, it has nothing to do with RTD. Rendering is happening within Sphinx.

Reply:
@samccann good idea WRT line numbering, I _always_ forget the periods :)